apps/dashboard/app/views/widgets/_sessions.html.erb

```diff
@@ -0,0 +1,27 @@
+<%-
+  max_sessions = @user_configuration.active_sessions[:max_sessions] || 3
+  poll_delay = @user_configuration.active_sessions[:poll_delay] || 10000
```

Comment:
I'm wondering if we need these. The second can use the same `ENV['POLL_DELAY']` and the first... maybe just default to showing only 3.

I'm just worried about the explosion of options. ( `ENV['POLL_DELAY']` will need to be refactored to be a part of the configuration_singleton either way).

Reply:
OK - will update....